packages/plugins/typescript/mongodb/src/visitor.ts

```diff
@@ -23,6 +22,7 @@ import {
   InterfaceTypeDefinitionNode,
   UnionTypeDefinitionNode,
 } from 'graphql';
+import { wrapTypeNodeWithModifiers } from '../../../other/visitor-plugin-common/src/utils';
```

Comment:
Please fix this import, it should take it from `@graphql-codegen/visitor-plugin-common`